sidekick/lib/src/init/init_command.dart

```diff
@@ -67,6 +67,15 @@ class InitCommand extends Command {
     if (!isValidCliName(cliName)) {
       throw invalidCliNameErrorMessage;
     }
+    bool cliNameCollides = false;
+
+    which(cliName).paths.forEach((element) {
+      cliNameCollides = !element.contains('.sidekick');
```

Comment:
Can you explain whats the intention behind the `.sidekick` check?

Reply:
If we want to regenerate an existing Sidekick Project, like after an Major Update of Sidekick, it would crash because the CLI name already exists. So I excluded the `.sidekick` directory.